src/HPRT/toolkit/models.py

```diff
@@ -112,25 +111,20 @@ def get_labels(self):
         labels = {}
 
         json_data = open(os.path.join('static', "questions.json"), 'r')
-        htq_list = json.load(json_data)[0]['questions']
+        htq_list = json.load(json_data)[0:3]
+        # HTQ questions
         i = 1
-        for index in htq_list:
+        for index in htq_list[0]['questions']:
```

Comment:
I think you should use the enumerate function here as in 

for i, index in enumerate(htq_list[0]['questions']):

This will help you to not need to reset i each time